CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Release pipeline now also publishes source distributions
```

Comment:
I think this should really go to "Added" because it's a purely incremental thing. Or to put it differently: otherwise, we could put everything under 'changed', because we will always 'change' files :D

Reply:
changed (or should I say added :D)